tests/test_concepts.py

```diff
@@ -149,3 +149,22 @@ def test_literal():
     assert Orex().literal("cat").is_match(s)
 
     assert not Orex().literal("rat").is_match(s)
+
+
+def test_orex_and():
+    s = "foo123bar"
+    assert Orex().literal("foo").orex_and("bar").is_match(s)
```

Comment:
Ah, now I see that and is a unary operator. That a bit unintuitive, no?
So it would take literally all the string before as part of the and?

I would think that 
```
Orex().orex_and('foo', 'bar')
```
or 
```
Orex().orex_and('foo', Orex().literal('bar'))
```
would be the standard way.

Reply:
Another question, how do we feel about shortening Orex to ox? I think that would reduce the clutter quite a bit. A bit unfortunate that we cannot get away from the priming (), unless we define some completely new operators like the rebus guy did.

Reply:
### First comment

Both work actually. I have modified the unit tests to reflect it and have included the cases you designed.

For **Orex().orex_and("foo", "bar")**, the expression is (?=.*(?=.*)(?=.*foo))(?=.*bar). Basically, it takes the empty expression in Orex and runs and on it with foo, so we have (?=.*)(?=.*foo); then, we run and on it with bar, so it is matching (?=.*)(?=.*foo), which composes to (?=.*(?=.*)(?=.*foo)) and bar, so they all compose to (?=.*bar).

You can even do the and operation on a completely new instance. As an example, Orex().orex_and("foo").orex_or("bar") creates (?=.*foo)|bar, which is the same as foo|bar.

### Second comment

Mixed feelings. If we do it that way we need to reorder the function names: ox_or sounds a bit too close to xor, clashing with the readability objective. The alternative or_ox and and_ox is not as bad but not ideal either.